lib/webpacker/railtie.rb

```diff
@@ -15,4 +15,10 @@ class Webpacker::Engine < ::Rails::Engine
     Webpacker.bootstrap
     Spring.after_fork { Webpacker.bootstrap } if defined?(Spring)
   end
+
+  initializer "webpacker.logger" do
+    config.after_initialize do |app|
+      Webpacker.logger = ::Rails.logger
```

Comment:
If simply using `Webpacker.logger = ...` in an initializer, you're right I don't think it would work since the Rails initializer will execute before this `config.after_initialize` block. But, I believe it would be possible to wrap the Rails initializer call in a separate `config.after_initialize` block. 

As indicated in an [earlier comment](https://github.com/rails/webpacker/pull/625#discussion_r132017784), this approach appears to be used elsewhere in the Rails, e.g., https://github.com/rails/rails/blob/master/activestorage/lib/active_storage/engine.rb#L16.